sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/TransactionalBatchTest.java

```diff
@@ -0,0 +1,379 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.HttpConstants;
+import com.azure.cosmos.implementation.ISessionToken;
+import com.azure.cosmos.implementation.guava25.base.Function;
+import com.azure.cosmos.models.CosmosItemResponse;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import org.assertj.core.data.Offset;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Test;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+
+import static com.azure.cosmos.implementation.batch.BatchRequestResponseConstant.MAX_OPERATIONS_IN_DIRECT_MODE_BATCH_REQUEST;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TransactionalBatchTest extends BatchTestBase {
+
+    private CosmosClient batchClient;
+    private CosmosContainer batchContainer;
+    private CosmosAsyncContainer batchAsyncContainer;
```

Comment:
you are mixing the async hierarchy with non-async hierarchy.

the usage expectation is once a user decide async vs non-async with stick to that decision everywhere and never mixes the usage of the two.

As these tests might get used by the end user as sample code we should ensure we are following above guideline.

Reply:
Ack. Have removed that test case.